### PR TITLE
UI adjustments

### PR DESCRIPTION
**Pull request by nekhaly**
This PR should cover most of the comments about the UI and its responsiveness.

### CODE REVIEW

File: frontend/src/App.vue
Comment (manuelwedler) [inline frontend/src/App.vue:2]: I think another breakpoint where decrease the padding of the logo container might be useful for a full hd screen. At the moment a user might still need to scroll

**Review by manuelwedler — COMMENTED** [review]
Did you check another time with Olympe if the breakpoints and other changes are fine? If so this should be fine

**Comment by nekhaly** [comment]
> Did you check another time with Olympe if the breakpoints and other changes are fine? If so this should be fine

I will have another review shot with her today. We agreed on most of the changes already, but didn't make a final review after merging with your approach!

**Comment by manuelwedler** [comment]
Perfect :)

**Comment by manuelwedler** [comment]
There also seems to be a problem with the CI. I tries to run `yarn lint` in the root folder and not in the `frontend` folder.

File: frontend/src/App.vue
Comment (nekhaly) [reply frontend/src/App.vue]: I made a re-adjustment to avoid the scrolling and reviewed it with @olympe-silve. So everything is good with the current state

**Review by manuelwedler — APPROVED** [review]
Perfect :) Ready for the release

**Pull request merged**
